keyboards/dekunukem/duckypad/keymaps/m4cs/keymap.c

```diff
@@ -21,24 +21,30 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include QMK_KEYBOARD_H
 
 #include "stdio.h"
+#include "raw_hid.h"
 #include <unistd.h>
```

Comment:
```suggestion
```
I don't think this header is needed.

Reply:
Wouldn't this be needed since I'm using `raw_hid_receive` ?

Reply:
I don't see where raw HID requires unistd.h.

Reply:
Oh my bad I read the diff incorrectly